src/sinks/datadog/metrics.rs

```diff
@@ -159,17 +159,21 @@ impl HttpSink for DatadogSink {
 
         http::Request::get(self.uri.clone())
             .header("Content-Type", "application/json")
+            .header("DD-API-KEY", self.config.api_key.clone())
```

Comment:
Were we not doing anything with the API key for non-healthcheck requests before...?

Reply:
Ah, we're still putting it in the URI on 118.